applications/emocam/emocam1.js

```diff
@@ -84,11 +86,19 @@ var webWindow;
     var propList = ["headRotation", "headType"];
     handlerId = MyAvatar.addAnimationStateHandler(function (props) {
         if (Date.now() - lastDataArrived < 2000) {
+            if (isProceduralblinkingEnabled) {
```

Comment:
the lastDataArrived being offset in the past and this creates confusion

Reply:
This part means that if there's no data arriving for 2 seconds we should not use the positions and just pass through animation data. Maybe we should have a comment there?

Reply:
maybe just make it a constant, for greater clarity and in case any tuning is needed?

Reply:
Done!